sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScanBase.scala

```diff
@@ -1047,10 +1070,12 @@ class MultiFileParquetPartitionReader(
     // Dump parquet data into a file
     dumpDataToFile(dataBuffer, dataSize, splits, Option(debugDumpPrefix), Some("parquet"))
 
+    val includeColumn = toCudfColumnNames(readDataSchema, clippedSchema,
```

Comment:
Nit: This should probably be a pluralized name (e.g. `includeColumns`) since it's a collection of columns.  Applies to other places where this code was copied.

Reply:
Done